src/Seaborn.jl

```diff
@@ -1,47 +1,5 @@
-__precompile__(true)
 module Seaborn
 
-export
-despine,
-lineplot,
-scatterplot,
-relplot,
-jointplot,
-pairplot,
-distplot,
-kdeplot,
-rugplot,
-lmplot,
-regplot,
-residplot,
-interactplot,
-coefplot,
-factorplot,
-catplot,
-boxenplot,
-boxplot,
-violinplot,
-stripplot,
-swarmplot,
-pointplot,
-barplot,
-countplot,
-heatmap,
-clustermap,
-tsplot,
-palplot,
-FacetGrid,
-PairGrid,
-JointGrid,
-axes_style,
-set_style,
-plotting_context,
-set_context,
-set_color_codes,
-reset_defaults,
-reset_orig,
-load_dataset
-
 using Reexport
```

Comment:
Is `Reexport` still being used?

Reply:
I left the `@reexport PyPlot` line in there